ci/tasks/generate-prometheus-test-jq.yml

```diff
@@ -0,0 +1,11 @@
+platform: linux
+inputs:
+  - name: input
+outputs:
+  - name: output
+run:
+  path: sh
+  args:
+    - -euxc
+    - |
+      echo ".last_successful_config == $(jq '.prometheus_config_etag.value' input/terraform-outputs.json)" > output/test.jq
```

Comment:
I guess `jq` default quoting for outputted strings makes the `$(...)` here work, which is interesting...

Reply:
yes i thought it was quite neat